client/src/pages/404.tsx

```diff
@@ -0,0 +1,26 @@
+// import React from "react"
+// import { useNavigate } from "react-router-dom"
+import Wind from "assets/logos/Wind.png"
+import Button from "components/generic/FigmaButtons/FigmaButton"
+
+const NoMatch = () => {
+  const handleBack = () => {
+    window.history.go(-1)
```

Comment:
we should navigate back to home page, also should use `react-router` functions for this instead (prefer to redirect to home)